test/MerklePatriciaProof.js

```diff
@@ -0,0 +1,115 @@
+const MerklePatriciaProofMock = artifacts.require("./MerklePatriciaProofMock.sol");
+const proofJson =  require("./data/proof.json");
+const ethUtil = require ("ethereumjs-util");
```

Comment:
Are we using `ethereumjs-uti` package for any thing other than sha3 ? If not then would suggest to use sha3 available in web3.